### ISSUE TITLE
Viewing Inactive Cameras

### ISSUE THREAD

**Issue by GarryGrizzly555** [ParadiseSS13/Paradise#12919]
**Issue Description**:

You can view deactivated cameras like normal if you check the camera and re-enter the camera console again.

**What did you expect to happen**: 

To see a black screen, or something else that prevents me from seeing what the deactivated camera is looking at.

**What happened instead**:

I saw what the deactivated camera is looking at.

**Why is this bad/What are the consequences**:

Could be abused to see things on broken or otherwise deactivated cameras  (Not sure what exactly counts as deactivated) that could sometimes make the antags job harder.

**Steps to reproduce the problem**:

1. Deactivate a camera.
2. Walk up to a camera console (In this case the Syndi Lavaland Outpost one)
3. Check the deactivated camera.
4. Exit the console and enter it again.
5. You should see what the camera is looking at despite it being deactivated.

**When did the problem start happening**:

23/01/20
Basically today in Poland as I am writing this report.
Should still work.

**Extra information**:

I'm not sure if it's an issue for one specific console, but it should be possible on all of them. 
I did that on the Syndicate Lavaland Outpost.

**Comment by moxian** [comment]
Cannot reproduce; Was likely fixed by https://github.com/ParadiseSS13/Paradise/pull/13750
